examples/hello_world/run

```diff
@@ -2,6 +2,11 @@
 set -e
 set -x
 
+OAK_MANAGER_ADDRESS=${OAK_MANAGER_ADDRESS:-"127.0.0.1:8888"}
+
 cargo build --release --target=wasm32-unknown-unknown --manifest-path=examples/hello_world/Cargo.toml
-bazel run //examples/hello_world/client -- \
+bazel run //examples/hello_world/client \
```

Comment:
Maybe we should have a single top-level example runner script that takes the example name (`hello_world`, `machine_learning`, etc.) as parameter?